src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -23,6 +24,7 @@
 
   <ItemGroup>
     <PackageReference Include="LibLog" Version="5.0.6" PrivateAssets="All" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.1" />
```

Comment:
Should this also have `PrivateAssests="All"`?

Reply:
Yep, fixed.

Reply:
Is this needed for the trace id generation? I'm not sure Datadog will accept this change because it then exposes an OOB dependency on .NET Framework, unless we did some sort of IL Repacking (if that would even work for this Unsafe library). Some of the functionality is built-in on .NET Core, so removing this NuGet package and doing a conditional compilation of whatever API's are built-in to .NET Core could work